frontend/src/store/config.js

```diff
@@ -206,6 +207,57 @@ export const useConfigStore = defineStore('config', () => {
     return state.value?.defaultNodesCIDR ?? '10.250.0.0/16'
   })
 
+  const shootAdminKubeconfig = computed(() => {
+    return state.value?.shootAdminKubeconfig
+  })
+
+  const allShootAdminKubeconfigExpirations = [
+    {
+      title: '10m',
+      value: 600,
+    },
+    {
+      title: '30m',
+      value: 1800,
+    },
+    {
+      title: '1h',
+      value: 3600,
+    },
+    {
+      title: '3h',
+      value: 10800,
+    },
+    {
+      title: '6h',
+      value: 21600,
+    },
+    {
+      title: '12h',
+      value: 43200,
+    },
+    {
+      title: '1d',
+      value: 86400,
+    },
+    {
+      title: '3d',
+      value: 259200,
+    },
+    {
+      title: '7d',
+      value: 604800,
+    },
+  ]
+  const shootAdminKubeconfigExpirations = computed(() => {
+    return filter(allShootAdminKubeconfigExpirations, ({ value }) => {
+      if (!shootAdminKubeconfig.value?.maxExpirationSeconds) {
```

Comment:
This condition does not depend on the value. It whould be evaluated outside of the loop.